CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ S3 BACKEND:
 * Most S3 compatible remote state backends should now work without checksum errors / 400s ([#821](https://github.com/opentofu/opentofu/issues/821))
 * Logging has been re-added to S3 remote state calls ([#821](https://github.com/opentofu/opentofu/issues/821))
 * Fixed validation for certain optional fields ([875](https://github.com/opentofu/opentofu/issues/875))
+* Fix access to known references when using a import block for module resources ([#1105](https://github.com/opentofu/opentofu/pull/1105))
```

Comment:
we probably want this in bugfixes and not s3 backend

Reply:
Fixed